task-2/html/index.js

```diff
@@ -0,0 +1,121 @@
+const questions = [
+  {question : 'Выберите собаку',
+    answer :{
+      a : {
+        label : 'Вы ответили верно!',
+        image : 'images/blackpuppy.jpg'
+      },
+      b : {
+        label : 'Попробуйте ещё раз',
+        image : 'images/whitepuppy.jpg'
+      },
+    },
+    correct :'A'
+  },
+  {question : 'Чем обработать собаку от клещей?', 
+    answer :{
+      a : {
+        label : 'Вы ответили верно!',
+        image : 'images/bravecto.jpg'
+      },
+      b : {
+        label : 'Попробуйте ещё раз',
+        image : 'images/bars.jpg'
+      },
+    },
+    correct :'A'
+  },
+  {question : 'Каким кормом кормить собаку?',
+    answer :{
+      a : {
+        label : 'Попробуйте ещё раз',
+        image : 'images/pedigree.jpg'
+      },
+      b : {
+        label : 'Вы ответили верно!',
+        image : 'images/grandorf.jpg'
+      },
+    },
+  correct :'B'
+  }
+] 
+
+let questionIndex = 0;
+let correctAnswer = questions[questionIndex]['correct'];
+
+
+showQuestion(questionIndex);
+
+
+
+function showQuestion(){
+document.getElementById('question').innerHTML = questions[questionIndex]['question'];
+document.getElementById('image-A').setAttribute('src',questions[questionIndex]['answer'].a.image) ;
+document.getElementById('image-B').setAttribute('src',questions[questionIndex]['answer'].b.image) ;
+correctAnswer = questions[questionIndex]['correct'];
+};
+
+let container = document.getElementById('container');
+container.onclick = function(event){
+  let target = event.target;
+  let answer = target.dataset.answer;
+  if (target.className != "answer") return;
+  if(answer == correctAnswer){
```

Comment:
Let's move 63-71 to separate function - onRightAnswerClick, and 73-75 to onWrongAnswerClick